src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -538,7 +538,7 @@ class SpfxDoctorCommand extends BaseProjectCommand {
     },
     '1.18.0': {
       gulpCli: {
-        range: '^1 || ^2',
+        range: '^1 || ^2 || ^3',
         fix: 'npm i -g gulp-cli@2'
```

Comment:
If we support v3, then in the recommendation, we should suggest installing the latest version. Let's update this

Reply:
Ah that makes sense, thank you for the review. I've updated the PR.